client/webpack.server.rails.config.js

```diff
@@ -8,13 +10,17 @@ module.exports = {
   output: {
     filename: 'server-bundle.js',
     path: '../app/assets/javascripts/generated',
-
-    // CRITICAL for enabling Rails to find the globally exposed variables.
-    libaryTarget: 'this',
   },
   resolve: {
     extensions: ['', '.webpack.js', '.web.js', '.js', '.jsx', 'config.js'],
   },
+  plugins: [
```

Comment:
Should we have a common setting for non-hot-reload? I think this node setting is duplicated in client and server. Maybe this is just fine as is.